connectors/servicenow/src/test/java/com/vmware/connectors/servicenow/ServiceNowControllerTest.java

```diff
@@ -74,26 +74,26 @@ void testRegex() throws Exception {
         List<String> expected = Arrays.asList(
                 "REQ0010001",
                 "REQ0010003",
-                "REQ0010000", // not ideal
+//                "REQ0010000", // Should not match due to too many numbers
                 "REQ0010005",
                 "REQ0010007",
                 "REQ0010009",
-                "REQ0010010", // not ideal
+                "REQ0010010", // Should not match due to trailing "X"
```

Comment:
Somehow, I didn't finish commenting these out and didn't notice that I hadn't re-run the build since an earlier version of the code.

I had @rworsnop go ahead and revert instead of me rushing out the fix, so I can verify manually again (in case I fooled myself on which version of the code I tested in my vm).